src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -56,12 +56,6 @@ public ITaskItem[] Analyzers
             get { return (ITaskItem[])_store["Analyzers"]; }
         }
 
-        public ITaskItem[] AnalyzerDependencies
```

Comment:
Don't you need to update the targets files too: https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L73